src/transformDimension.ts

```diff
@@ -61,9 +61,19 @@ export function transformDimension(token: DesignToken): DesignToken['value'] {
       transformed = transformProp(transformed, 'width');
       break;
     }
-    default:
+    default: {
       transformed = transformed as number | string;
       transformed = _transformDimension(transformed);
```

Comment:
Putting this code in the `default` case means only dimension tokens are supported for multi value but not for border / shadow / typography properties such as fontSize, borderWidth, shadow blur etc. dimensions.

What I think you need to do is put go through the `_transformDimension` function which is called in all switch cases, but basically add another wrapper function.

In purely pseudo code it would look something like:

```js
function transformDimensionWrapperTwo(tokenValue) {
  // the thing that actually adds the "px" unit when needed
  if (!isNaN(dim as number) && dim !== '' && parseFloat(dim as string) !== 0) {
    return `${dim}px`;
  }
  return `${dim}`;
}

function transformDimensionWrapperOne(tokenValue) {
  // if the tokenValue is a string with spaces, split it up into separate dimension values
  return tokenValue.split(' ')..map(transformDimensionWrapperTwo).join(' ');
}

function transformDimension(token) {
  switch(token.type) {
    case 'whatever case': {
      token.value = transformDimensionWrapperOne(token.value)
      break;
    }
  }
}
```

This way you're correctly layering things, where:
- the switch is the first layer that checks the token type and selects the correct subproperty for object values or just the value "as is" in the default "dimension" type case
- the multi-value splitting happens in the second layer
- the actual adding of the 'px' happens in the third layer

I would add some test cases for typography/border/shadow type tokens to ensure that the multi-value px adding is supported there as well, in the current state of this PR such a test would fail because only dimension values in "dimension" tokens is handled